deps/jerry/jerry-core/api/jerry.c

```diff
@@ -3629,6 +3630,26 @@ jerry_stop_cpu_profiling (void)
 #endif
 }
 
+void
+jerry_heap_profiler_take_snapshot (const char *path)
+{
+  jerry_assert_api_available ();
+
+#ifndef JERRY_HEAP_PROFILER
+  JERRY_UNUSED (path);
```

Comment:
If `JERRY_HEAP_PROFILER` not defined, we should not do operate on any file handle.

Reply:
#define JERRY_UNUSED(x) ((void) (x))
JERRY_UNUSED does nothing on it's param. It just suppresses compiler warning "define but not used".